package.json

```diff
@@ -169,8 +169,5 @@
   "peerDependencies": {
     "react": "^16.3.0",
     "react-dom": "^16.3.0"
-  },
-  "resolutions": {
-    "create-react-context": "0.2.2"
```

Comment:
`react-popper` now defines max version of `create-react-context`

https://github.com/FezVrasta/react-popper/releases/tag/v1.3.1